examples/using-web-socket/go.mod

```diff
@@ -0,0 +1,105 @@
+module github.com/gofr-dev/gofr/examples/using-web-socket
+
+go 1.22.3
+
+require (
+	github.com/gorilla/websocket v1.5.0
+	github.com/stretchr/testify v1.9.0
+	gofr.dev v1.8.1
```

Comment:
it should use a replace statement, which would refer to the local version of gofr.